pipelines/utilities/run_longhorn_e2e_test.sh

```
@@ -39,6 +39,7 @@ run_longhorn_e2e_test(){
   fi
 
   yq e -i 'select(.spec.containers[0] != null).spec.containers[0].env[6].value="'${LONGHORN_TEST_CLOUDPROVIDER}'"' ${LONGHORN_TESTS_MANIFEST_FILE_PATH}
+  yq e -i 'select(.spec.containers[0] != null).spec.containers[0].env[7].value="'${TF_VAR_arch}'"' ${LONGHORN_TESTS_MANIFEST_FILE_PATH}
```

Shellcheck (0.10.0)

[warning] TF_VAR_arch is referenced but not assigned.

(SC2154)